llvm/lib/Target/BPF/BTFDebug.cpp

```diff
@@ -707,7 +707,7 @@ void BTFDebug::visitArrayType(const DICompositeType *CTy, uint32_t &TypeId) {
   // Visit array element type.
   uint32_t ElemTypeId;
   const DIType *ElemType = CTy->getBaseType();
-  visitTypeEntry(ElemType, ElemTypeId, false, false);
+  visitTypeEntry(ElemType, ElemTypeId, false, false, false);
```

Comment:
Indeed, now we have three bool parameters. Probably we should have a struct to replace these bool parameters now.